Do something with the parsed return message

pepijndeboever opened the pull request:
Fixes #14

No exit call because the diagnostic message wouldn't be printed.

pepijndeboever committed:
Do something with the parsed return message
Fixes #14

No exit call because the diagnostic message wouldn't be printed.

pepijndeboever committed:
Do something with the parsed return message
Fixes #14

No exit call because the diagnostic message wouldn't be printed.

driesmp committed:
Fix typo and make English

driesmp committed:
Make FormatOutput static

driesmp committed:
Make the function FormatOutput actually static

driesmp committed:
Order functions alphabetically

driesmp commented on dynamicprefixvici.c:
```diff
@@ -98,6 +101,12 @@ int main(int argc, char** argv)
+                // Let the program exit with 1 if vici was unable to add the pool
+                if(parsed_message[0] != 'y')
+                {
+                    error_encountered = true;
```
Should we also error_encountered = true; at line 95, if(response == NULL)

driesmp replied:
Actually, why don't we just have a print statements in the if statement and exit(1) right after?
We could also do this in other locations like at line 95, we print the error, but don't exit(1) yet.

if(parsed_message[0] != 'y'){
print  what we need to
exit(1)
}

driesmp committed:
Update dynamicprefixvici.c

driesmp committed:
Elaborate comment of default value

When adding larger pools than /97, strongSwan will still round them to that number, wont error out.

driesmp committed:
Clean up headers

driesmp committed:
Remove math lib

driesmp committed:
Sort headers and group based on /usr/lib or /usr/local/lib

driesmp committed:
Remove blannk line

driesmp committed:
Update README

driesmp committed:
Update README

driesmp reviewed (changes requested):
if(parsed_message[0] != 'y'){
print what we need to
exit(1)
}

driesmp committed:
change function variable name in FormatOutput

driesmp committed:
The function definition too

driesmp committed:
Comments too

driesmp committed:
One more

driesmp committed:
Update README.md

driesmp committed:
Merge .sh files in a Makefile

driesmp committed:
Update README after Makefile addition

driesmp committed:
Expand Makefile

driesmp committed:
tab everything

driesmp committed:
more tabs

driesmp committed:
remove math library link

driesmp committed:
Add clean target to Makefile

driesmp committed:
Be more descriptive in variables and function names

driesmp committed:
CMakeLists.txt: streamline  with project name

driesmp committed:
Order alphabetically

driesmp committed:
Simplify debug printing

driesmp committed:
Remove redundant print with address_poool

driesmp committed:
Reorder vici_init and add a comment

driesmp committed:
Eleborate comment when not able to parse

driesmp committed:
Remove obsolete comment

driesmp committed:
Remove Status in print statements

driesmp committed:
Use strcat over sprintf

driesmp committed:
Revert ccdcbbf

driesmp committed:
Test something

driesmp committed:
Revert 8a91967

driesmp committed:
Be consistent with capitalizing

driesmp commented:
Should probably also try to parse errmsg to print when succes = no

![image](https://user-images.githubusercontent.com/32487486/230634636-7234da91-b466-4377-a894-7a1ce1751e70.png)

driesmp committed:
Consistency and additional comments

driesmp committed:
Be more descriptive in fucntion name

driesmp committed:
Use puts over printf when printing a literal string

driesmp committed:
Puts already appends a newline character

driesmp committed:
Dont drop puts

pepijndeboever committed:
Do something with the parsed return message
Fixes #14

driesmp committed:
Drop boolean logic

driesmp committed:
Drop boolean logic

driesmp commented:
Tried to rebase the branch but kinda messed it up, my bad.
Manually committed in 732efed.
